src/commands/flow/internal/allFlowsCompleted.ts

```diff
@@ -0,0 +1,34 @@
+export const allFlowsCompleted = (
+  flows: any
+): {
+  completed: boolean
+  completedWithAllSuccess: boolean
+} => {
+  const flowNames = Object.keys(flows)
+
+  let jobsCount = 0
+  let jobsWithSuccessStatus = 0
+  let jobsWithNotSuccessStatus = 0
+
+  flowNames.map((name) => (jobsCount += flows[name].jobs.length))
```

Comment:
In my opinion, `reduce` operator would be a better choice for the job

Reply:
fixed